packages/headless/src/controllers/core/did-you-mean/headless-core-did-you-mean.ts

```diff
@@ -95,6 +110,9 @@ export interface DidYouMeanState {
  * @param engine - The headless engine.
  * @param props - The configurable `DidYouMean` controller properties.
  * @returns A `DidYouMean` controller instance.
+ *
+ * @group Core controllers
+ * @category DidYouMean
  */
 export function buildCoreDidYouMean(
   engine: CoreEngine,
```

Comment:
Is there a way for this controller to not be documented ? It is not exported in the index.ts files. Same with most of the stuff in core/.

Reply:
I just realized that `buildCoreDidYouMean` isn't exported, so my tags here are superfluous. I'll need to do a pass in general to remove unnecessary tags.

I do think I need to export the state and controller interfaces though, because they're referenced by both the Search and Insight buildDidYouMean functions...